### PR TITLE
[MINOR] Code refactor Bookie env check during start.

### PR DESCRIPTION
**Pull request by WJL3333**
[MINOR] code refactor Bookie env check during start.

move bookie environment check logic code to `BookieEnvirnmentChecker`
this may makes code much clearer.

no logic change or method delete.

### CODE REVIEW

**Comment by eolivelli** [comment]
@WJL3333 thanks for your patch.

is this a preparatory patch for some other contribution or is it a code cleanup without any other follow up work ?

**Comment by WJL3333** [comment]
@eolivelli thanks for reply. this patch is just cleanup refactor, no more follow up work.

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/environment/checker/BookieEnvironmentChecker.java [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+        for (File journalDirectory : journalDirectories) {
+            masterCookie.writeToDirectory(journalDirectory);
+        }
+        for (File dir : allLedgerDirs) {
+            masterCookie.writeToDirectory(dir);
+        }
+        masterCookie.writeToRegistrationManager(rm, conf, version);
+    }
+
+
```
Comment (dlg99) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/environment/checker/BookieEnvironmentChecker.java:333]: nit: formatting: unnecessary empty lines

**Review by dlg99 — APPROVED** [review]
lgtm

**Review by fpj — COMMENTED** [review]
I don't see much value in moving this code to a separate package (in fact it forces at least a couple of methods to become public), but otherwise, for organization purposes, it sounds fine to organize in the way proposed.

**Comment by WJL3333** [comment]
@fpj yes, this is just for organize propose~

**Comment by eolivelli** [comment]
@fpj do you think we can go ahead an merge this patch ?

**Comment by Ghatage** [comment]
@reddycharan can you PTAL?

**Comment by StevenLuMT** [comment]
fix old workflow,please see #3455 for detail

**Pull request closed without merging**